packages/block-library/src/site-logo/edit.js

```diff
@@ -427,9 +423,9 @@ export default function LogoEdit( {
 
 	let alt = null;
 	if ( mediaItemData ) {
-		alt = mediaItemData.alt;
-		if ( logoUrl !== mediaItemData.url ) {
-			setLogoUrl( mediaItemData.url );
+		alt = mediaItemData.alt_text;
```

Comment:
Should we check if `alt_text` exists before using it? 🤔

Reply:
Good catch, `mediaItemData` can be `undefiend` while we're fetching the item.

P.S. I think this Edit component could use minor refactoring but decided to leave that for another P.R.

Reply:
Hmmm you're right. Worst case scenario if there is no alt text, `mediaItemData.alt_text` will be `undefined` without any errors. 👍